boards/arm/mimx8mm_phyboard_polis/mimx8mm_phyboard_polis.dts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2022 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx8mm_m4.dtsi>
+#include "mimx8mm_phyboard_polis-pinctrl.dtsi"
+
+/ {
+	model = "Phyboard Polis NXP i.MX8M Mini";
+	compatible = "nxp,mimx8mm_phyboard_polis";
+
+	aliases {
+		uart-4 = &uart4;
+		uart-3 = &uart3;
+		uart-2 = &uart2;
+		uart-1 = &uart1;
+		led0 = &led_blue;
+	};
+
+	chosen {
+		zephyr,flash = &tcml_code;
+		zephyr,sram = &tcmu_sys;
+		zephyr,console = &uart4;
+		zephyr,shell-uart = &uart4;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_leds>;
```

Comment:
Why is `pinctrl` needed here? The `gpio_pin_configure` API should be able to select GPIO mode when it is used to configure pins. If this isn't working, please feel free to open an issue against this SOC

Reply:
You are completely right. I'll remove the ```pinctrl``` for the leds.